FiftyOne.Pipeline.Core/FlowElements/PipelineBuilder.cs

```diff
@@ -384,9 +428,11 @@ private void AddElementToList(
             }
 
             // Add the element to the list.
```

Comment:
Cosmetics: this comment is no longer accurate, as we are using dict, not list, also it was too obvious anyway - I'd just remove comments that do not add much info.